freqtrade/freqtradebot.py

```diff
@@ -173,6 +173,14 @@ def _create_pair_whitelist(self, pairs: List[str]) -> List[Tuple[str, str]]:
         """
         return [(pair, self.config['ticker_interval']) for pair in pairs]
 
+    def get_free_open_trades(self):
+        """
+        Return the number of free open trades slots or 0 if
+        max number of open trades reached
+        """
+        open_trades = len(Trade.get_open_trades())
+        return max(0, self.config['max_open_trades'] - open_trades)
```

Comment:
Nice call - i assume a negative number can happen if max_open_trades is modified to a lower number (from 5 to 3) after 5 trades are already open?

Reply:
Yes, for example in this case.